app/Http/Controllers/EntriesController.php

```diff
@@ -79,16 +79,50 @@ public function add(Request $request)
 
     public function push(Request $request)
     {
+
         $validatedData = $request->validate([
-            'id' => 'required|exists:entries',
+            'id' => 'required|array',
         ]);
-        try {
-            $entry = Entry::findOrFail($validatedData['id']);
-            $entry->push();
-            return redirect('/panel/entries/list')->with('status', 'Entrada Validada Manualmente!');
-        } catch (Exception $e) {
-            return redirect('/panel/entries/list')->with('status', 'Erro ao validar entrada!');
+
+        foreach(request()->get('id') as $id){
+            try {
+                $entry = Entry::findOrFail($id);
```

Comment:
Please move this logic to an `EntryService` or `EntryRepository` class, then inject said class into this controller method and run the `update()` method from here.